Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -251,6 +258,7 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
 
   internal func configureWith(value: Project) {
     self.viewModel.inputs.configureWith(project: value)
+    self.viewModel.inputs.current(configuration: AppEnvironment.current.config)
```

Comment:
We don't need this input, `AppEnvironment` is globally accessible so we can just access it where needed in the view model.